src/frontend/src/lib/components/receive/ReceiveAddresses.svelte

```diff
@@ -148,6 +107,55 @@
 		testId={RECEIVE_TOKENS_MODAL_ETH_SECTION}
 	>
 		{$i18n.receive.ethereum.text.ethereum}
+
+		<span slot="notes" class="text-secondary text-sm"
+			>{$i18n.receive.icp.text.your_private_eth_address}</span
+		>
+	</ReceiveAddressWithLogo>
+
+	<div class="mb-6">
+		<Hr />
+	</div>
+
+	<ReceiveAddressWithLogo
```

Comment:
Am I safe to assume that there were no code changes, only order changed here (minus the new sentence)?

Reply:
Yes I think so